gitpod-io/gitpod
# Allow configuration of s3 region for registry backend

## Description

This PR adds an option to configure the bucket region. Alongside it also ensures the secret is created even when the registry is set to `inCluster` since s3 backend can be set for `inCluster` registry.

## Related Issue(s)

Fixes #9800

## How to test

Use `S3` as the `In-cluster Storage provider` in KOTS config UI. You will see the option to set the `Region` for S3

## Release Notes

```release-note
[kots] Add option to set the `S3` bucket region used for registry backend
```

## Documentation
